boards/arm/nrf51_ble400/Kconfig.defconfig

```diff
@@ -0,0 +1,33 @@
+# Kconfig - nRF51 BLE400 board configuration
+#
```

Comment:
missing copyright here.

And please squash all your commits and send one commit to this PR... git push -f

Reply:
Done everything